recipes-mender/mender/mender_0.1.bb

```diff
@@ -45,6 +45,11 @@ do_compile() {
   export GOPATH
   PATH="${B}/bin:$PATH"
   export PATH
+  DEFAULT_CERT_MD5="e034532805e44b9125b443c32bbde581"
+
+  if [ $(md5sum $MENDER_CERT_LOCATION | awk '{ print $1 }') = $DEFAULT_CERT_MD5 ]; then
```

Comment:
ah, yes, i usually use shellcheck to remind me about this, but the plugin was turned on for .bb files